packages/bigquery/src/table.js

```diff
@@ -826,7 +826,7 @@ Table.prototype.getRows = function(options, callback) {
  * //-
  * var metadata = {
  *   encoding: 'ISO-8859-1',
- *   sourceFormat: 'NEWLINE_DELIMITED_JSON'
+ *   format: 'JSON'
```

Comment:
@stephenplusplus Thank you to review.

I wrote

```js
      table.import(files, {
        sourceFormat: 'NEWLINE_DELIMITED_JSON',
        schema: {
          fields: [...]
        },
        writeDisposition: 'WRITE_EMPTY'
      }, done)
```

like this.

https://github.com/e-jigsaw/google-cloud-node/blob/5d1c04f22ce48a0365854da656ea1a95a368f8a8/packages/bigquery/src/table.js#L870

This line assign `NEWLINE_DELIMITED_JSON` to `format` and then

https://github.com/e-jigsaw/google-cloud-node/blob/5d1c04f22ce48a0365854da656ea1a95a368f8a8/packages/bigquery/src/table.js#L872

This line returns `undefined` because `FORMATS` is

https://github.com/e-jigsaw/google-cloud-node/blob/5d1c04f22ce48a0365854da656ea1a95a368f8a8/packages/bigquery/src/table.js#L40-L44

Finally it is configured by `csv`. Then I caught error.

Is this unexpected behavior?